dashboard/src/t5gweb/utils.py

```diff
@@ -24,10 +24,9 @@ def email_notify(ini, blist):
     msg = EmailMessage()
     msg.set_content(body)
 
-    msg["Subject"] = ini["subject"]
+    msg["Subject"] = ini[subject] if subject else ini["subject"]
     msg["From"] = ini["from"]
-    msg["to"] = ini["to"]
-
+    msg["to"] = ini[recipient] if recipient else ini["to"]
```

Comment:
while you are poking around in there, looks like we were inconsistent with variable capitalisation. can you make that `msg["To"]` assuming that doesn't break anything?

Reply:
Surprised it works lowercase, in the [documentation](https://docs.python.org/3/library/email.examples.html) it's uppercase...just changed it anyways